caffe2/serialize/crc_alt.h

```diff
@@ -101,8 +101,11 @@ uint32_t crc32_16bytes_prefetch(const void* data, size_t length, uint32_t previo
   // Windows always little endian
   #define __BYTE_ORDER __LITTLE_ENDIAN
 
+  #if !defined(_M_ARM64)
```

Comment:
@gaborkertesz-linaro Here you guard against using Intel intrinsics on Windows on Arm, MSVC has no specific Arm intrinsics to handle prefetching?

Reply:
According to Microsoft documentation, it's an MSVC definition for ARM64:
https://docs.microsoft.com/en-us/cpp/preprocessor/predefined-macros?view=msvc-170#microsoft-specific-predefined-macros
>  Defined as 1 for compilations that target 64-bit ARM processors. Otherwise, undefined.

Here we need the __BYTE_ORDER definition for MSVC, but for ARM64 we shouldn't include the Intel specific xmmintrin.h.
I think this change is the less intrusive and has been tested with MSVC.

Do you have any other better proposal?